--- data/config.in
+++ data/config.in
@@ -26,8 +26,8 @@
 		<dllentry dll="__Internal" name="MoveMemory" target="mono_win32_compat_MoveMemory"/>
 		<dllentry dll="__Internal" name="ZeroMemory" target="mono_win32_compat_ZeroMemory"/>
 	</dllmap>
-	<dllmap dll="gdiplus" target="@prefix@/lib/libgdiplus@libsuffix@" os="!windows"/>
-	<dllmap dll="gdiplus.dll" target="@prefix@/lib/libgdiplus@libsuffix@"  os="!windows"/>
-	<dllmap dll="gdi32" target="@prefix@/lib/libgdiplus@libsuffix@" os="!windows"/>
-	<dllmap dll="gdi32.dll" target="@prefix@/lib/libgdiplus@libsuffix@" os="!windows"/>
+	<dllmap dll="gdiplus" target="libgdiplus@libsuffix@" os="!windows"/>
+	<dllmap dll="gdiplus.dll" target="libgdiplus@libsuffix@"  os="!windows"/>
+	<dllmap dll="gdi32" target="libgdiplus@libsuffix@" os="!windows"/>
+	<dllmap dll="gdi32.dll" target="libgdiplus@libsuffix@" os="!windows"/>
 </configuration>
